widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -174,7 +181,12 @@ public void onItemRangeInserted(int positionStart, int itemCount) {
             int totalItemCount = adapter.getItemCount();
             int lastIndex = totalItemCount - 1;
             if (isInBottom) {
-                chatRecyclerView.scrollToPosition(lastIndex);
+                RecyclerView.ViewHolder holder = chatRecyclerView.findViewHolderForAdapterPosition(lastIndex);
+                if (holder instanceof WebViewViewHolder) {
+                    postDelayed(() -> chatRecyclerView.scrollToPosition(lastIndex), 100);
```

Comment:
Either a comment would be useful or extract the delay constant to a self descriptive constant. Do I understand correctly that delay is used to give time for the web view to load?

Reply:
Yes, a `WebView` needs time for rendering.